### DISCUSSION TITLE
Add Thorium 3 to WinGet

### DISCUSSION THREAD

**Discussion by N-H-U-A-C** [edrlab/thorium-reader#2676] [Developer corner]
Hi,

First of all, thanks for your hard work.

Is it possible to add Thorium 3 to Windows Package Manager? Currently the only available version is the 2.4.2.

**Comment by danielweck** [comment]
hello, I wonder who maintains the WinGet repository? not EDRLab as far as I know.

**Comment by danielweck** [comment]
this:

 https://winstall.app/apps/EDRLab.Thorium

https://wingetgui.com/apps/EDRLab-Thorium

https://www.winget.me/search/thorium

**Comment by N-H-U-A-C** [comment]
Oh, I didn't realize you're not the one behind the WinGet repo.
